src/httpation_uri_tests.erl

```diff
@@ -0,0 +1,97 @@
+-module(httpation_uri_tests).
```

Comment:
The point here is that hand-written tests are limited by our imagination and patience. Machine generated tests are limited by the accuracy of our property specification and the compute time we are willing to give the test generator.